src/py/flwr/client/app.py

```diff
@@ -263,6 +265,31 @@ def _load_client_app() -> ClientApp:
         transport, server_address
     )
 
+    run_tracker = _RunTracker()
```

Comment:
I note that if we don't explicitly do:
```
del run_tracker
```
at the end of `_start_client_internal()`, the function will never exit after a `ctrl+c` (it will hang). I tested this by having just a `SuperLink` and a single `flower-client-app` connected.

Reply:
Thanks! I just updated it